lib/operations/mongo_client_ops.js

```diff
@@ -595,11 +600,4 @@ function validOptions(options) {
   }
 }
 
-function validOptionsLowerCaseToCamelCase() {
-  validOptionNames.reduce((obj, name) => {
-    obj[name.toLowerCase()] = name;
```

Comment:
Not related to this PR per se: Since I believe we do this a lot, do you think it is worth it to write our own impl of [`_.mapValues`](https://lodash.com/docs/4.17.10#mapValues)?

Reply:
¯\_(ツ)_/¯  where else do we use it? It seems pretty easy to me to write these in raw javascript, and it's pretty clear what's going on. I think we'd save a few characters that way?